software/apps/baremetal/Makefile

```diff
@@ -34,7 +32,7 @@ all_llvm: $(ALL_LLVM)
 $(APPS): % : $(BIN_DIR)/% $(APPS_DIR)/Makefile $(shell find $(RUNTIME_DIR)/**.{S,c,h,ld} -type f)
```

Comment:
I think here is the dependence in the Makefile that generates all of the data together even if it not be used in the target kernel. But low-priority to change.